apps/editing-toolkit/editing-toolkit-plugin/dotcom-fse/lib/site-intent/use-site-intent.js

```diff
@@ -1,27 +1,14 @@
-import apiFetch from '@wordpress/api-fetch';
-import { useState, useEffect, useCallback } from '@wordpress/element';
-
 // FIXME: We can use `useSiteIntent` from `@automattic/data-stores` and remove this.
 // https://github.com/Automattic/wp-calypso/pull/73565#discussion_r1113839120
 const useSiteIntent = () => {
```

Comment:
Should we deprecate this hook as well, or not?
https://github.com/Automattic/wp-calypso/blob/trunk/packages/data-stores/src/queries/use-site-intent.ts#L11

Reply:
I think we have to implement it on Jetpack repo or [switch to another endpoint](https://github.com/Automattic/wp-calypso/pull/77627). But maybe we can simply return the intent from the options of the site settings.

Reply:
> Should we deprecate this hook 

Im not entirely sure if that would work in this situation or not (in the editor), but maybe is more robust with wpcom-proxy-request under the hood where this hook was simply api-fetch. Id imagine that hook works fine for atomic in calypso?

> I think we have to implement it on Jetpack repo or https://github.com/Automattic/wp-calypso/pull/77627. But maybe we can simply return the intent from the options of the site settings.

That other endpoint you link lives in jetpack-mu-wpcom which fits pretty well for most cases, but I held off as it seems like the GET permissions need to be updated for the use we wanted here in the editor.  Reading intent from options where they exist is an option as well.

Reply:
Yes, I know. But people might feel confused about why we need to get the site intent from the launchpad endpoint as they're different features 😓

Reply:
> Im wondering if we should put an inline script in jetpack-mu-wpcom...

If site_intent is used across multiple places, I think that makes sense! There's no need for each individual feature to re-request this data, and adding it as a JS global server-side looks like the most efficient/most performant way to get the data.